pkg/activator/revision_test.go

```diff
@@ -110,14 +110,13 @@ func TestActiveEndpoint_Reserve_ReadyTimeoutWithError(t *testing.T) {
 		ch <- a.ActiveEndpoint(testNamespace, testRevision)
 	}()
 
-	<-time.After(100 * time.Millisecond)
 	select {
 	case <-ch:
 		t.Error("Unexpected result before revision is ready.")
-	default:
+	case <-time.After(100 * time.Millisecond):
+		break
```

Comment:
This break is not needed I think? There's no `for` around the `select`

Reply:
It's not, it's more for readability, I think it generates nop here. Removed.